### ISSUE TITLE
investment insight news search for S+P 500 returns dubious links

### ISSUE THREAD

**Issue by southernseas** [IBM-Bluemix/investment-insights-for-asset-managers#110]
Ran the new analysis for S+P 500 ... returns some very shady non pc news articles.
Needs additional filters in the news search to just get financial articles e.g. NLU taxonomy /finance

**Comment by Twanawebtech** [comment]
I agree discovery results can be improved to return more related articles. 
I labeled this as a future enhancement for now. We can look into this in the future.
